lib/rules/no-useless-call.js

```diff
@@ -71,25 +71,35 @@ function isValidThisArg(expectedThis, thisArg, context) {
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
-    return {
-        CallExpression: function(node) {
-            if (!isCallOrNonVariadicApply(node)) {
-                return;
-            }
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow unnecessary calls to `.call()` and `.apply()`",
+            category: "Best Practices",
+            recommended: false
+        },
+
+        schema: []
```

Comment:
Sorry, if I'm late, but diff should looks much better with fixes like:
```js
exports.meta = {...};
exports.create = function(context) { ...
```
intead of assigning new object:
`module.exports = { meta: ..., create: ... };`.